packages/cli/README.md

```diff
@@ -325,6 +326,28 @@ DESCRIPTION
   Displays environment variables that can be used to deploy apps and app extensions.
 ```
 
+## `shopify app execute`
```

Comment:
Let's make sure this description notes that the command doesn't currently do anything.